doc/cephadm/certmgr.rst

```diff
@@ -183,9 +192,8 @@ To retrieve the content of a specific certificate:
 
    ceph orch certmgr cert get <certificate_name> [--service_name <value>] [--hostname <value>] [--no-exception-when-missing]
 
-Replace `<certificate-name>` with the actual certificate name from `ceph
-orch certmgr cert ls`. For certificates with host or service scope, include
-the `--hostname` or `--service_name` arguments as needed.
+Use `<service-name>` as listed by `ceph orch certmgr bindings ls`. For certificates with host or service scope,
+include the `--hostname` or `--service_name` arguments as needed.
```

Comment:
In other places of the document (and docs in general) these are using double backticks instead of single backtick.

```suggestion
Use ``<service-name>`` as listed by ``ceph orch certmgr bindings ls``. For certificates with host or service scope,
include the ``--hostname`` or ``--service_name`` arguments as needed.
```